client/my-sites/customer-home/main.jsx

```diff
@@ -153,18 +139,12 @@ const Home = ( {
 		);
 	}
 
-	// While fetching all the data, show placeholder.
-	// After fetching all the data:
-	//  - For eCommerce & eCommerce trial, don't show placeholder.
-	//  - For Woo Express Trial, Essential & Performance, show placeholder while redirecting to WooCommerce Home (search for `wc-admin` under customer-home/controller.jsx).
+	// Ecommerce Plan's Home redirects to WooCommerce Home, so we show a placeholder
+	// while doing the redirection.
 	if (
 		isSiteWooExpressEcommerceTrial &&
 		( isRequestingSitePlugins || hasWooCommerceInstalled ) &&
-		( fetchingJetpackModules || ssoModuleActive ) &&
-		( isRequestingSitePlans ||
-			isRequestingSitePurchases ||
-			isSiteWooExpress ||
-			purchase?.isWooExpressTrial )
+		( fetchingJetpackModules || ssoModuleActive )
```

Comment:
Hmm, I don't think we want to revert those changes. Otherwise we WooCommerce loading placeholder will be displayed during site creation for Entrepreneur trial.

Reply:
Oh actually it's just the WP logo sign showing to hide what's behind (which is My Home), so this applies for both Entrepreneur Trial & WooExpress Trial (which doesn't exist anymore) so it is actually fine.

Reply:
I see. Thanks for the context update. I will test this out tomorrow to see the logo for myself. I did not notice it before. But I have no concerns over this change now. 🙇

Reply:
I was actually questioning this for some time yesterday as well when reviewing the PR. 🙂